src/test/java/org/snomed/snowstorm/validation/DroolValidationServiceTest.java

```diff
@@ -0,0 +1,192 @@
+
+        index++;
+        assertEquals(Severity.WARNING, invalidContents.get(index).getSeverity());
+        assertEquals("Active FSN should end with a valid semantic tag.", invalidContents.get(index).getMessage());
+
+        index++;
+        assertEquals(Severity.WARNING, invalidContents.get(index).getSeverity());
+        assertEquals("A concept's semantic tags should be compatible with those of the active parents.", invalidContents.get(index).getMessage());
+    }
+
+    @Test
+    void testValidateConceptWithoutRelationship() throws ServiceException {
+        Concept foundConcept = conceptService.find("100001", DEFAULT_BRANCH);
+        assertNotNull(foundConcept);
+        foundConcept.getDescriptions().stream().forEach(description -> description.setAcceptabilityMap(Collections.singletonMap(Concepts.US_EN_LANG_REFSET,
+                Concepts.descriptionAcceptabilityNames.get(PREFERRED))));
+
+        final Concept updatedConcept = conceptService.update(foundConcept, DEFAULT_BRANCH);
+
+        assertEquals(0, updatedConcept.getRelationships().size());
+        List <InvalidContent> invalidContents = droolValidationService.validateConcept(DEFAULT_BRANCH, updatedConcept);
+        assertEquals(4, invalidContents.size());
+
+        int index = 0;
+        assertEquals(Severity.WARNING, invalidContents.get(index).getSeverity());
+        assertEquals("Test resources were not available so assertions like case significance and US specific terms checks will not have run.", invalidContents.get(index).getMessage());
+
+        index++;
+        assertEquals(Severity.WARNING, invalidContents.get(index).getSeverity());
+        assertEquals("Active FSN should end with a valid semantic tag.", invalidContents.get(index).getMessage());
+
+        index++;
+        assertEquals(Severity.ERROR, invalidContents.get(index).getSeverity());
+        assertEquals("Active concepts must have at least one IS A relationship.", invalidContents.get(index).getMessage());
+
+        index++;
+        assertEquals(Severity.WARNING, invalidContents.get(index).getSeverity());
+        assertEquals("A concept's semantic tags should be compatible with those of the active parents.", invalidContents.get(index).getMessage());
+    }
+
+    @Test
+    void testValidateConceptAgainstConcreteValue() throws ServiceException {
+        Concept foundConcept = conceptService.find("100001", DEFAULT_BRANCH);
+        assertNotNull(foundConcept);
+
+        foundConcept.getDescriptions().stream().forEach(description -> description.setAcceptabilityMap(Collections.singletonMap(Concepts.US_EN_LANG_REFSET,
+                Concepts.descriptionAcceptabilityNames.get(PREFERRED))));
+
+        foundConcept.addRelationship(new Relationship("100002", ISA, SNOMEDCT_ROOT).setCharacteristicTypeId(Concepts.STATED_RELATIONSHIP));
```

Comment:
We no longer use stated relationships although the Snowstorm DroolValidationService will convert the fake "relationships" within axioms into stated DroolsRelationship objects for validation. 
In order to use this stuff in the test please:
- change this one to an inferred relationship
- also add an axiom to the concept with a relationship within that
Thanks!

Reply:
The unit tests have been updated @kaicode